built_value/lib/standard_json_plugin.dart

```diff
@@ -5,18 +5,25 @@ import 'dart:convert' show JSON;
 
 /// Switches to "standard" JSON format.
 ///
-/// The default serialization format is more powerful, supporting polymorphism
-/// and more collection types. But, you may need to interact with other systems
-/// that use simple map-based JSON. If so, use [SerializersBuilder.addPlugin]
-/// to install this plugin.
+/// The default serialization format is more powerful, with better performance
+/// and support for more collection types. But, you may need to interact with
+/// other systems that use simple map-based JSON. If so, use
+/// [SerializersBuilder.addPlugin] to install this plugin.
 class StandardJsonPlugin implements SerializerPlugin {
   static final BuiltSet<Type> _unsupportedTypes =
       new BuiltSet<Type>([BuiltSet, BuiltListMultimap, BuiltSetMultimap]);
 
+  /// The field used to specify the value type if needed. Defaults to `$`.
+  final String discriminator;
```

Comment:
Differentiator ? Slightly less bias.

Reply:
:D

I agree it's a bit aggressive sounding, but this appears to be the standard term

https://www.google.ch/search?q=json+discriminator